lnrpc/rpc.proto

```diff
@@ -797,18 +829,31 @@ message SendCoinsResponse {
     string txid = 1 [json_name = "txid"];
 }
 
+message ListUnspentRequest {
+	/// The minimum number of confirmations to be included.
+	int32 min_confs = 1;
+
+	/// The maximum number of confirmations to be included.
+	int32 max_confs = 2;
+}
+message ListUnspentResponse {
+	/// A list of utxos
+	repeated Utxo utxos = 1 [json_name = "utxos"];
+
+}
+
 /** 
 `AddressType` has to be one of:
 
 - `p2wkh`: Pay to witness key hash (`WITNESS_PUBKEY_HASH` = 0)
 - `np2wkh`: Pay to nested witness key hash (`NESTED_PUBKEY_HASH` = 1)
 */
-message NewAddressRequest {
-    enum AddressType {
+enum AddressType {
```

Comment:
Will this be a breaking RPC change?

Reply:
I don't think so. It tests fine on the lncli command line; I also just tested it over grpc using a python client and it seemed to work fine.

Reply:
It will break usages of `NewAddressRequests` for projects importing `lnrpc`, but I think that's fine. It is only a simple name change.